src/gps/GPS.cpp

```diff
@@ -18,8 +18,8 @@
 #if defined(NRF52840_XXAA) || defined(NRF52833_XXAA) || defined(ARCH_ESP32)
 HardwareSerial *GPS::_serial_gps = &Serial1;
 #elif defined(ARCH_RASPBERRY_PI)
-// need a translation layer to make _serial_gps work with pigpio https://abyz.me.uk/rpi/pigpio/cif.html#serOpen
-HardwareSerial *GPS::_serial_gps = NULL;
+
+HardwareSerial *GPS::_serial_gps = &Serial1;
 #else
```

Comment:
duplicate line, can merge into one, no need for the elif

Reply:
Agreed, fixed.